source/module_basis/module_ao/parallel_orbitals.h

```diff
@@ -13,23 +13,23 @@ class Parallel_Orbitals : public Parallel_2D
 
     /// local size of bands, used for 2d wavefunction
     /// must divided on dim1 because of elpa interface
-    int ncol_bands;
-    int nrow_bands;
+    int ncol_bands=0;
```

Comment:
initialize the variables in the constructor

Reply:
The variable has been putted in the constructor